scripts/windows-testing/convert-symlinks.mjs

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
As I read through this file, it makes sense and I am good with it, but any debugging I think would require quite a bit of rereading to remember the usage and context of this.

Since this is going to be relatively load bearing and hopefully rarely looked at, I think we should over comment the specifics throughout this. Particularly in what the windows specific commands are that we don't use day to day too much.